Project.toml

```diff
@@ -20,13 +20,4 @@ TaylorSeries = "6aa5eb33-94cf-58f4-a9d0-e4b2c4fc25ea"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-Coverage = "0.9.3, 1.0"
-DifferentialEquations = "6.8.0"
-DocStringExtensions = "0.8.1"
-ForwardDiff = "0.10.6"
-HCubature = "1.4.0"
-Optim = "0.19.4, 0.20"
-PyPlot = "2.8.2"
-SpecialFunctions = "0.8.0"
-TaylorSeries = "0.10.1"
-julia = "1.1"
+julia = "~1.6"
```

Comment:
We can update the compat entries, but we probably shouldn't remove them. Also, using `julia = "1.1"` should be fine (it's permissive of version 1.1 and above).